### PR TITLE
chore(test): some fixes for e2e tests

### LINKED ISSUE #8953: Fix k8s e2e tests that are failing on cicd
### Bug description

Currently all e2e tests that check k8s functionalities are broken on cicd and need to be fixed.

### Operating system

Windows 11

### Installation Method

Installer from website/GitHub releases

### Version

next (development version)

### Steps to reproduce

_No response_

### Relevant log output

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by cbr7**
### What does this PR do?
Fixes some flaky behavior in e2e tests.

### What issues does this PR fix or reference?
https://github.com/containers/podman-desktop/issues/8953

### CODE REVIEW

File: tests/playwright/src/utility/operations.ts [hunk 1]
```diff
@@ -185,7 +185,7 @@ export async function deletePodmanMachine(page: Page, machineVisibleName: string
     .poll(async () => await resourcesPage.resourceCardIsVisible(RESOURCE_NAME), { timeout: 10000 })
     .toBeTruthy();
   const podmanResourceCard = new ResourceConnectionCardPage(page, RESOURCE_NAME, machineVisibleName);
-  await playExpect(podmanResourceCard.providerConnections).toBeVisible({ timeout: 10_000 });
+  await playExpect(podmanResourceCard.providerConnections).toBeVisible({ timeout: 10000 });
```
Comment (benoitf) [inline tests/playwright/src/utility/operations.ts]: I think it's more readable using the grouping feature `10_000` vs `10000`

so I would keep this convention

Comment (cbr7) [reply tests/playwright/src/utility/operations.ts]: done, changed.

File: tests/playwright/src/specs/podman-machine-dashboard-onboarding.spec.ts [hunk 2]
```diff
@@ -31,6 +31,7 @@ test.skip(
 );
 
 test.beforeAll(async ({ runner, welcomePage, page }) => {
+  test.setTimeout(120000);
```
Comment (jeffmaury) [inline tests/playwright/src/specs/podman-machine-dashboard-onboarding.spec.ts]:
```suggestion
  test.setTimeout(120_000);
```

File: tests/playwright/src/specs/podman-machine-onboarding.spec.ts [hunk 3]
```diff
@@ -51,6 +51,7 @@ test.skip(
 );
 
 test.beforeAll(async ({ runner, welcomePage, page }) => {
+  test.setTimeout(120000);
```
Comment (jeffmaury) [inline tests/playwright/src/specs/podman-machine-onboarding.spec.ts]:
```suggestion
  test.setTimeout(120_000);
```

**Review by jeffmaury — APPROVED** [review]
LGTM

File: tests/playwright/src/model/pages/details-page.ts [hunk 4]
```diff
@@ -49,7 +51,9 @@ export abstract class DetailsPage extends BasePage {
 
   async activateTab(tabName: string): Promise<this> {
     const tabItem = this.tabs.getByRole('link', { name: tabName, exact: true });
-    await playExpect(tabItem).toBeVisible();
+    await waitUntil(async () => await tabItem.isVisible(), {
```
Comment (odockal) [inline tests/playwright/src/model/pages/details-page.ts:54]: Note: waitUntil has only 5 sec default timeout while toBeVisible has 10 sec I think.

Comment (cbr7) [reply tests/playwright/src/model/pages/details-page.ts:54]: No, both are 5 seconds by default.

**Review by odockal — APPROVED** [review]

File: tests/playwright/src/model/pages/details-page.ts (hunk 4 above)
Comment (odockal) [reply tests/playwright/src/model/pages/details-page.ts:54]: ok then.

**Review by benoitf — APPROVED** [review]

**Pull request merged**